eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -94,6 +94,10 @@ jobs:
       - ${{ if eq(parameters.jobParameters.isFullMatrix, false) }}:
         - OSX.1014.Amd64.Open
         - OSX.1015.Amd64.Open
+    
+    # OSX arm64
+    - ${{ if eq(parameters.platform, 'OSX_x64') }}:
```

Comment:
This is a typo, the check should be for arm64 not x64

Reply:
Thanks.  I always do something like that ;-)